spec/index.bs

```diff
@@ -4286,6 +4287,12 @@ dictionary GPUTextureViewDescriptor
     ::
         The dimension to view the texture as.
 
+    : <dfn>usage</dfn>
+    ::
+        The allowed {{GPUTextureUsage|usage(s)}} for the texture view. Must be a subset of the
+        {{GPUTexture/usage}} flags of the texture. Defaults to the full set of {{GPUTexture/usage}}
+        flags of the texture.
```

Comment:
Hmm, should we error if you request some viewFormat that has zero overlap with the usage of the texture?

Reply:
Not sure I understand this?

Reply:
Example:
`{ format: 'rgba8unorm', viewFormats: ['rgba8unorm-srgb'], usage: GPUTextureUsage.STORAGE_BINDING }`
This makes it impossible to actually use that viewFormat so should it be an error to create this texture?

Reply:
Similar to how we disallow usage from being `0`.

Reply:
Ah, got it. So something like (obvious psuedocode) `usage & viewFormat.allowedUsages != 0`. Makes sense.

Reply:
TBH I'm not sure that this is needed, it seems to add preemptive validation just because it might be an interesting warning for developers. Maybe it should just be a warning in this case. IIRC there are other cases where we don't error even if the operation makes things that cannot be used.

Reply:
After discussion in the WG this needs to be changed to a TODO.